### DISCUSSION TITLE
Is this a bug?

### DISCUSSION THREAD

**Discussion by xfqwdsj** [chrisbanes/haze#112] [Q&A] [answered]
I just noticed a phenomenon from [documentation](https://chrisbanes.github.io/haze/) that seems to be a problem (?) The edges of a component with a blur effect applied will have the color of the component immediately next to it. A closer look reveals that haze still blurs the edges as the scrollable content scrolls to the end.

![image](https://github.com/chrisbanes/haze/assets/37076189/b037d9bc-2e8e-4356-8195-e0e07dffac29)

Thanks!

**Comment by chrisbanes** [comment, marked as answer]
Yep, opened #114
